### ISSUE TITLE
Demo Issue: Properly Format Hyperlinks

### ISSUE THREAD

**Issue by david-boles** [duckietown/docs-duckiesky_high_school#33]
... explanation of how we want to format hyperlinks ....

- [ ] A.1.1
- [x] A.1.2
- [ ] A.1.3
- [ ] A.2.1
- [ ] A.2.2
- [ ] A.2.3
- etc...

**Comment by peytonstrong** [comment]
Oh ok I get how the check-off thing could definitely be useful. So should we assign a checklist for every individual author for each lesson?

**Comment by david-boles** [comment]
@peytonstrong Yup, for sure. I don't actually know if someone else can check things off the checklist I made (can you?) but that's maybe either fine if you wanted to double check people's changes or they could make the issues for their units themselves.
